src/app.pcss

```diff
@@ -53,7 +54,7 @@ body {
     --radius: 0.5rem;
   }
 
-  .dark {
+  /* .dark {
```

Comment:
Couldn't find a better way to force the light theme 🤔 🤦‍♂️

Reply:
I used `setMode` from `mode-watcher` somewhere in the code 🤔